src/hotspot/share/classfile/classLoader.hpp

```diff
@@ -373,7 +373,9 @@ class ClassLoader: AllStatic {
 #endif
 
   static char* lookup_vm_options();
-  static bool has_jvmci_module();
+
+  // Determines if the `module_name` module is resolvable.
+  static bool is_module_resolvable(const char* module_name);
```

Comment:
Is "resolvable" the right concept here? Or should it be something like "findable" instead?

Reply:
Assuming --limit-modules isn't used, it is testing if the module is "observable".

Reply:
I assume this function should therefore be named `is_module_observable`?

Reply:
How about this:
```
// Determines if the named module is present in the
// modules jimage file or in the exploded modules directory.
static bool is_module_observable(const char* module_name);
```

Reply:
That looks okay to me. I don't have any other comments on this part.